### ISSUE TITLE
Contentinfo landmark should not be nested

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10164]
## Description
In the CMS, there are messages that appear at the top of the View and Edit screens (Content lock, broken links, etc.).  These messages are contained in a contentinfo landmark which is a top level landmark and should not be nested.

This also occurs on the login page with the error message on the form.

## Additional Context
I found 2 related Drupal Core issues:
- https://www.drupal.org/project/drupal/issues/2942404
- https://www.drupal.org/project/drupal/issues/3159933

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/184181588-4cad9bda-b764-4181-904c-ee972e7ff59f.png)

![image](https://user-images.githubusercontent.com/106678594/184217735-2d4e5b18-7594-406f-93e6-9221f9dac805.png)

## Accessibility Standard

WCAG version 2.0 A, [Criterion 1.3.1](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

## Acceptance Criteria
- [ ] Review Drupal issues and determine if they are in fact referring to the same elements this ticket is
- [ ] Determine path forward for remediation

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by EWashb** [comment]
Removing "needs refining"-- forgot to do this.

**Comment by swirtSJW** [comment]
Both patches in the two issues seem viable.  The first issue is a little more robust so if it will apply and work, that is the better choice https://www.drupal.org/project/drupal/issues/2942404#comment-14628932

**Comment by laflannery** [comment]
@swirtSJW Should I make a new task for implementation of the first patch?

**Comment by swirtSJW** [comment]
I don't think we need a new ticket. We can just use this one.

**Comment by swirtSJW** [comment]
I applied this patch from [https://www.drupal.org/project/drupal/issues/2942404](https://www.drupal.org/project/drupal/issues/2942404#comment-14628932)  and ended up with broken styes
![image](https://user-images.githubusercontent.com/5752113/224379229-5a897327-4b45-4893-bcdd-6a6cfd097a4e.png)

![image](https://user-images.githubusercontent.com/5752113/224379282-13e29525-0f88-49ac-8d19-1d4cb58a0c4e.png)

**Comment by swirtSJW** [comment]
The existing patch from this issue https://www.drupal.org/project/drupal/issues/3159933 does not apply 
https://www.drupal.org/files/issues/2021-10-26/3159933-15.patch

```
Cannot apply patch 3159933 - Contentinfo landmark (https://www.drupal.org/files/issues/2021-10-26/3159933-15.patch)!
```
